src/Dropdown.elm

```diff
@@ -117,19 +125,19 @@ type Key
     }
 
 -}
-init : String -> State item
+init : String -> State
 init id =
```

Comment:
Initially I considered including the default value as a parameter in the init command. I decided against that because the init is not invoked every time the dropdown is opened/ model is changed, and therefore you wouldn’t be able to refresh the default depending on the model e.g. in example 3 you wouldn’t be able to default the city based on what country was selected.